### PR TITLE
glew 1.13.0

### PR DESCRIPTION
**Pull request by dunn**
Another try at #43062.

### CODE REVIEW

**Comment by dunn** [comment]
@xpicox is this the error you were seeing with your test?
```
        Testing glew
==> Using the sandbox
/usr/bin/sandbox-exec -f /tmp/homebrew20151115-46106-1kzgdks.sb /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby -W0 -I /usr/local/Library/Homebrew -- /usr/local/Library/Homebrew/test.rb /usr/local/Library/Formula/glew.rb --verbose
==> /usr/local/Cellar/glew/1.13.0/bin/glewinfo
Error: glewCreateContext failed
Error: glew: failed
<0> expected but was
<1>.

File: Library/Formula/glew.rb [hunk 1]
```diff
@@ -28,6 +30,28 @@ def install
...
+        }
+        return 0;
+      }
+    EOS
+    system ENV.cc, "-framework", "OpenGL", "-framework", "GLUT",
+           "-lglew", testpath/"test.c", "-o", "test"
+    system "./test"
   end
 
   test do
```
Comment (xu-cheng) [inline Library/Formula/glew.rb]: Two test block? You may want to remote the old one.

**Comment by xpicox** [comment]
@dunn yes

File: Library/Formula/glew.rb (hunk 1 above)
Comment (dunn) [inline Library/Formula/glew.rb]: Ahhh, something weird happened with a rebase.  Thanks.

**Comment by mikemcquaid** [comment]
:+1:

**Comment by dunn** [comment]
@mikemcquaid did you mean to merge 71a42c3bdab574dbeec5814bd00e9bf5cda15d54?

**Comment by mikemcquaid** [comment]
@dunn no, reverted in ff5bb45

**Pull request closed without merging**